TodoApp/Views/Controllers/ItemListViewController.swift

```diff
@@ -9,16 +9,21 @@
 import UIKit
 import RealmSwift
 
+private enum ItemListVCIdentifierType {
+    static let cell = "Cell"
+    static let nib = "ItemListTableViewCell"
+    static let edit = "unwindByItemEdit"
+    static let itemEdit = "itemEdit"
+    static let segueId = "unwindByItemAdd"
+}
```

Comment:
・種別ごとに分離すると分かりやすそうに思いました
・privateなのでItemListVCという接頭辞は付けなくても良さそうです

```suggestion
private enum CellIdentifier {
    static let cell = "Cell"
}

private enum NibName {
    static let itemListTableViewCell = "ItemListTableViewCell"
}

private enum SegueIdentifier {
    static let edit = "unwindByItemEdit"
    static let itemEdit = "itemEdit"
    static let segueId = "unwindByItemAdd"
}
```

Reply:
確認遅くなりました！
レビューありがとうございます！！

Reply:
Identifierの細分化とenumの命名の修正を行いました。
確認お願いいたします。

Reply:
👍🏻